config/7.8.0/envs/rhdm-authoring-ha.yaml

```diff
@@ -116,10 +116,8 @@ others:
                     timeoutSeconds: 10
                   resources:
                     limits:
-                      cpu: "1000m"
```

Comment:
datagrid, i believe, is one object that isn't yet configurable with resource limits. so, we likely need to keep these settings for the datagrid statefulsets in rhdm/pam authoring-ha envs.

Reply:
This configuration is set to the statefulsets (and set for all pods). As DataGrid is RH MW image it share the same cct_module scripts and the configuration applied here should be also handle by DataGrid image (in a same way as we are configuring e.g. Kie Server image). Am I right @spolti ? 
@desmax74 I think you can easily check if limits are applied to pods, when you deploy the app and take a look into DataGrid pod log, there should be -Xmx=1024 (As we set there memory limit to 2Gi and defautlt JVM mem ratio is set to 50%)